ui/src/adapters/api/index.ts

```diff
@@ -3,6 +3,21 @@ import { z } from "zod";
 import { getStrictParser } from "src/adapters/parsers";
 import { Env } from "src/domain";
 
+export type Sorter = { field: string; order: "ascend" | "descend" };
```

Comment:
"ascend" and "descend" looks weird as they are verbs. Order is "ascendant" or "descendant".

Reply:
[It's what it's...](https://ant.design/components/table#api)
<img width="773" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/a3a0dee8-02d0-44c8-92db-47efee3fa1c3">